arnoldPlugins/gaffer.mtd

```diff
@@ -1021,6 +1049,7 @@
 		page STRING "Flake"
 	[attr pref_name]
 		page STRING "Flake"
+		gaffer.layout.activator STRING "flakeSpaceIsPref"
```

Comment:
Looks like we could also use a similar activator for `pref_name` on the standalone `flakes` shader. Actually, maybe there are a few more :

```
noise.parameters.pref_name
camera_projection.parameters.pref_name
flakes.parameters.pref_name
triplanar.parameters.pref_name
```

Reply:
Fixed in 97e17f87cb26497eb9884db7315638ff59570e75